bin/list

```diff
@@ -161,6 +161,34 @@ const renderDirectory = async dir => {
   return render(details)
 }
 
+
+
+// To enhance, one can take inspiration from https://github.com/jfhbrook/node-ecstatic/blob/fe17a5623c18319f82101b6549e9e2e526e03d1f/lib/ecstatic.js#L284
+const isNotModified = (req, etag, fileStats) => {
+  try {
+    // Etag comparison
+    const requestEtag = req.headers['if-none-match'];
+    if (requestEtag && (requestEtag === etag)) {
```

Comment:
There is no need to check whether the `requestEtag` is defined, because `etag` is never `undefined` nor `null`. Thus `undefined` is guaranteed to not equal whatever `etag` is.